src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -299,6 +299,23 @@ public override String ConfigurationName
             }
         }
 
+        /// <summary>
+        /// This parameter specifies the SSH subsystem to use for the remote connection.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true,
```

Comment:
Please put this in the "SSH Parameters" section.  This should also include the FilePathSSHHostParameterSet.